R/summary.R

```diff
@@ -40,17 +37,14 @@ col_paths <- function(x) {
   make_col_df(x, visible_only = TRUE)$path
 }
 
-
-#' Print Row/Col Paths Summary
+#' Print row/columm paths summary
 #'
-#' @param x an `rtable` object
+#' @param x (`TableTree`)\cr an `rtable` object.
 #'
-#' @export
-#' @return A data.frame summarizing the row- or column-structure of \code{x}.
-#' @examples
-#' library(dplyr)
```

Comment:
havent we removed all instances of library in examples?

Reply:
I don't think `dplyr` was ever removed - there's quite a few places where it is in the examples.